gensim/test/test_tfidfmodel.py

```diff
@@ -169,103 +171,100 @@ def test_consistency(self):
         docs = [corpus[1], corpus[2]]
```

Comment:
Please add a `self.` to `corpus` here and everywhere.

Reply:
I changed `self.corpus` to `corpus` because my calculations for expected_docs are based on the corpus given at the top of test_tfidfmodel.py, and `self.corpus` (based on test_data/test_corpus.mm) doesn't give matching results

Reply:
If `self.corpus` is not used anywhere I suggest removing it from the `setUp` method.
@menshikh-iv  please suggest the corpus which should be chosen.

Reply:
Looks good. Just one last step, since we are not using `testcorpus.mm` (defined in the `setUp` mehtod) can you replace it with `text`, `dictionary` and `corpus` defined at the top of the code.
After that it is ready to merge @menshikh-iv

Reply:
ping @PeteBleackley

Reply:
Please use either `self.corpus` or `corpus`, the redundancy is polluting the code.